### DISCUSSION TITLE
英文参考文献中会议的反斜杠//

### DISCUSSION THREAD

**Discussion by Forest216** [nju-lug/NJUThesis#201] [Q&A]
![image](https://user-images.githubusercontent.com/41437440/233376177-ffb2a62a-a7c3-4f83-bf64-b8fc45825f02.png)
题目和会议名之间的反斜杠//感觉特别宽，与后面IEEE/CVF中间的/格式不一致

**Comment by note286** [comment]
你试试`\renewcommand{\SlashFont}{\rmfamily}`.

**Reply by Forest216** [reply]
报了这个错`Command \SlashFont undefined`，是需要导什么包嘛

**Reply by note286** [reply]
那说明你需要更新包，`biblatex-gb7714-2015`。

**Reply by note286** [reply]
去年这个时候我和作者提的，CTAN上也早就更新了，你可以去围观一下： https://github.com/hushidong/biblatex-gb7714-2015/issues/144 。

**Reply by Forest216** [reply]
谢谢，我刚才google的时候看到这个issue了哈哈哈哈
